src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -61,24 +63,36 @@ const ChartsHolder = (props: {
   const icuUtilizationData =
     projection && projection.getDataset('icuUtilization').data;
 
+  const getChartSummarys = (projection: Projection) => {
+    return {
+      [ChartType.CASE_GROWTH_RATE]: projection.getLatestColumnValue('rtRange'),
+      [ChartType.HOSPITAL_USAGE]: projection.getLatestColumnValue(
+        'icuUtilization',
+      ),
+      [ChartType.POSITIVE_TESTS]: projection.getLatestColumnValue(
+        'testPositiveRate',
+      ),
+    };
+  };
+
   return (
     <>
       {!projection ? (
         <NoCountyDetail countyId={props.countyId} stateId={props.stateId} />
       ) : (
         <ChartContentWrapper>
           <LocationPageHeader projections={props.projections} />
-          <SummaryStats stats={[]} />
+          {projection.isInferred && <SummaryStats stats={getChartSummarys(projection)} />}
```

Comment:
this shouldn't gate on inferred, just on their being data in 
    const rt = this.primary.rt!;
    const testPositiveRate = this.primary.currentTestPositiveRate;
    const icuUtilization = this.primary.currentIcuUtilization;